.gitignore

```diff
@@ -1,4 +1,6 @@
+config_local.py
 config.local.py
```

Comment:
hmm. Why both?

Reply:
Because I changed `config.local.py` to `config_local.py` everywhere and didn't want people to accidentally commit secrets.